geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -30,15 +31,21 @@ object HBaseFeatureIndex extends HBaseIndexManagerType {
   override val CurrentIndices: Seq[HBaseFeatureIndex] =
     Seq(HBaseZ3Index, HBaseXZ3Index, HBaseZ2Index, HBaseXZ2Index, HBaseIdIndex, HBaseAttributeIndex)
 
-  val DataColumnFamily = Bytes.toBytes("d")
-  val DataColumnFamilyDescriptor = new HColumnDescriptor(DataColumnFamily)
+  val DataColumnFamily = new Text("d")
```

Comment:
could you revert these back to byte arrays? doesn't seem to be used as texts anywhere